Legacy/PlayFab/Multiplayer.api.json

```diff
@@ -314,6 +314,9 @@
         },
         {
           "name": "UkSouth"
+        },
+        {
+          "name": "SwedenCentral"
```

Comment:
We'll need to keep an eye on this. introducing a new enum to the service could break older sdk versions. We should still generate with the new value, though.

Reply:
Actually, I just took a look at a couple SDKs. We don't actually use this enum in requests or responses. Instead, we just use strings, so we're all good.